source/ios/AdaptiveCards/ADCIOSVisualizer/resources/sample.json

```diff
@@ -1,11 +1,12 @@
 {
 	"spacing": {
-		"small": 3,
-		"default": 8,
-		"medium": 20,
-		"large": 30,
-		"extraLarge": 40,
-		"padding": 10
+		"extraSmall":4,
```

Comment:
Here i have configured spacing with what is present in teams to align both of them